### PR TITLE
Fix issue #1687 (NPE in TestIssue1120)

### PR DESCRIPTION
**Pull request by bob0bob**
When run from the test chooser, TestIssue1120 in jme3-examples crashes:

The application's test field is null because it wasn't initialized. The initialization was placed in main(), which the test chooser never executes.

### CODE REVIEW

**Comment by stephengold** [comment]
I self-assigned the issue before I saw this PR, and now we have 2 fixes for #1687!

File: jme3-examples/src/main/java/jme3test/bullet/TestIssue1120.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
  */
 public class TestIssue1120 extends SimpleApplication {
 
-    private static TestIssue1120 test;
+//    private static TestIssue1120 test;
```
Comment (stephengold) [inline jme3-examples/src/main/java/jme3test/bullet/TestIssue1120.java:68]: Please delete the commented-out code. Then this will be ready for integration.

Comment (bob0bob) [reply jme3-examples/src/main/java/jme3test/bullet/TestIssue1120.java]: done

**Comment by stephengold** [comment]
Thanks for your help.
By the way ... best practice is to create a new Git branch name for each pull request.

**Comment by bob0bob** [comment]
This is what I did.  I created a branch for each pull request,  so I could continue working.

**Comment by stephengold** [comment]
Then I don't understand why GitHub was showing 19 commits in the branch. Ah well...

**Pull request merged**